--- a/Makefile	2020-03-22 23:45:53.000000000 +0800
+++ b/Makefile	2020-04-27 20:28:02.391864563 +0800
@@ -2,8 +2,8 @@
 GOBUILD_DIR = gobuild
 GOPKG_PREFIX = pkg.deepin.io/dde/api
 GOSITE_DIR = ${PREFIX}/share/gocode
-libdir = /lib
-SYSTEMD_LIB_DIR = ${libdir}
+LIBDIR = /lib
+SYSTEMD_LIB_DIR = ${LIBDIR}
 SYSTEMD_SERVICE_DIR = ${SYSTEMD_LIB_DIR}/systemd/system/
 GOBUILD = env GOPATH="${CURDIR}/${GOBUILD_DIR}:${GOPATH}" go build
 
@@ -38,7 +38,7 @@
     image-blur \
     image-blur-helper
 
-all: build-binary build-dev ts-to-policy
+all: ts-to-policy
 
 prepare:
 	@if [ ! -d ${GOBUILD_DIR}/src/${GOPKG_PREFIX} ]; then \
@@ -66,12 +66,12 @@
 build-binary: prepare $(addprefix out/bin/, ${BINARIES})
 
 install-binary:
-	mkdir -pv ${DESTDIR}${PREFIX}${libdir}/deepin-api
-	cp out/bin/* ${DESTDIR}${PREFIX}${libdir}/deepin-api/
+	mkdir -pv ${DESTDIR}${LIBDIR}/deepin-api
+	cp _bin/* ${DESTDIR}${LIBDIR}/deepin-api/
 
 	mkdir -pv ${DESTDIR}${PREFIX}/bin
-	cp out/bin/dde-open ${DESTDIR}${PREFIX}/bin
-	rm ${DESTDIR}${PREFIX}${libdir}/deepin-api/dde-open
+	cp _bin/dde-open ${DESTDIR}${PREFIX}/bin
+	rm ${DESTDIR}${LIBDIR}/deepin-api/dde-open
 
 	mkdir -pv ${DESTDIR}${PREFIX}/share/dbus-1/system.d
 	cp misc/conf/*.conf ${DESTDIR}${PREFIX}/share/dbus-1/system.d/
@@ -106,7 +106,8 @@
 
 install-dev: ${addprefix install/lib/, ${LIBRARIES}}
 
-install: install-binary install-dev
+install: install-binary
+
 
 clean:
 	rm -rf out/bin gobuild out
@@ -118,3 +119,8 @@
 
 test: prepare
 	env GOPATH="${CURDIR}/${GOBUILD_DIR}:${GOPATH}" go test -v ./...
+binaries:
+	@echo ${BINARIES}
+
+libraries:
+	@echo ${LIBRARIES}
